routes/clucks.js

```diff
@@ -0,0 +1,57 @@
+const path = require('path');
+const multer = require('multer');
+const express = require('express');
+const router = express.Router();
+const knex = require('../db');
+
+const UPLOADS_DIR = 'uploads';
+const upload = multer({dest: path.join(__dirname, '..', 'public', UPLOADS_DIR)});
+
+// PATH: /clucks/new VERB: GET Serves form for creating clucks
```

Comment:
These notes are very helpful, and I'm sure they helped you with the development of this project.